tests/system/src/test/java/test/com/sun/marlin/DashedRectTest.java

```diff
@@ -0,0 +1,217 @@
+            path1.setStrokeDashOffset(DASH_PH);
+
+            final ObservableList<Double> pDashes = path1.getStrokeDashArray();
+            pDashes.clear();
+            for (int i = 0; i < 100; i++) {
+                pDashes.add(19.333);
+            }
+
+            // Create 2nd path shape
+            final Path path2 = new Path();
+            path2.getElements().addAll(
+                    new MoveTo(5, 5),
+                    new HLineTo(MAX),
+                    new VLineTo(MAX),
+                    new HLineTo(5),
+                    new ClosePath()
+            );
+            path2.setFill(null);
+            path2.setStroke(Color.BLUE);
+            path2.setStrokeWidth(2);
+            path2.setStrokeDashOffset(DASH_PH);
+            path2.getStrokeDashArray().setAll(DASH_LEN);
+
+            Scene scene = new Scene(new Group(path1, path2));
+            stage.setScene(scene);
+            stage.setTitle("DashedRectTest");
+            stage.show();
+
+            launchLatch.countDown();
+        }
+    }
+
+    @BeforeClass
+    public static void setupOnce() {
+        // Start the Application
+        new Thread(() -> Application.launch(MyApp.class, (String[]) null)).start();
+
+        try {
+            if (!launchLatch.await(TIMEOUT, TimeUnit.MILLISECONDS)) {
+                throw new AssertionFailedError("Timeout waiting for Application to launch");
+            }
+
+        } catch (InterruptedException ex) {
+            AssertionFailedError err = new AssertionFailedError("Unexpected exception");
+            err.initCause(ex);
+            throw err;
+        }
+
+        assertEquals(0, launchLatch.getCount());
+    }
+
+    @AfterClass
+    public static void teardownOnce() {
+        Platform.exit();
+    }
+
+    @Test(timeout = 10000)
+    public void TestDashedPath() throws InterruptedException, IOException {
+
+        final int size = MAX * 2;
+
+        final WritableImage img = new WritableImage(size, size);
+
+        done = false;
+        Platform.runLater(() -> {
+            final SnapshotParameters sp = new SnapshotParameters();
+            sp.setViewport(new Rectangle2D(0, 0, size, size));
+
+            myApp.stage.getScene().getRoot().snapshot(sp, img);
+
+            signalDone();
+        });
+
+        waitDone();
+
+        // Check image on few pixels:
+        final PixelReader pr = img.getPixelReader();
+
+        // 10, 5 = blue
+        checkPixel(pr, 10, 5, BLUE_PIXEL);
+    }
+
+    boolean done;
+
+    synchronized void signalDone() {
+        done = true;
+        notifyAll();
+    }
+
+    synchronized void waitDone() throws InterruptedException {
+        while (!done) {
+            wait();
+        }
+    }
+
+    private static void checkPixel(final PixelReader pr,
+                                   final int x, final int y,
+                                   final int expected) {
+
+        final int rgb = pr.getArgb(x, y);
+        if (rgb != expected) {
```

Comment:
Have you run this on Mac? I will try it, but I note that we've had problems doing exact color comparisons on Mac with the color space conversion (although snapshot might be unaffected...I'll check).

Reply:
Answering my own question, this runs fine on Mac, so no need to worry.